### PR TITLE
Update benefit preferences FAQ and CTA links

### PR DESCRIPTION
**Pull request by cehsu**
## Description
These changes update the FAQ and CTA links for the benefit preferences.
Resolves: https://github.com/department-of-veterans-affairs/vets-website/pull/9241#issuecomment-447028433

## Testing done
-verified locally

## Acceptance criteria
- [x] Update CTA and FAQ links

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by erikphansen — APPROVED** [review]

File: src/applications/personalization/preferences/helpers.jsx [hunk 1]
```diff
@@ -422,7 +424,8 @@ export const benefitChoices = [
       'We may be able to help you buy or build a home, or repair or refinance your current home. If you have a service-connected disability, you may want to consider applying for a grant to help you make changes to your home that will help you live more independently. ',
     cta: {
       description: housingCTADescription,
-      link: '/health-care',
+      link:
+        'https://www.ebenefits.va.gov/ebenefits/about/feature?feature=cert-of-eligibility-home-loan&_ga=2.26749749.986087660.1544627138-757629982.1539696635',
```
Comment (erikphansen) [inline src/applications/personalization/preferences/helpers.jsx]: does this `_ga` need to be a part of the URL?

Comment (cehsu) [reply src/applications/personalization/preferences/helpers.jsx]: Great catch, removed.

**Comment by Samara-Strauss** [comment]
Can you let me know when these have merged so I can do another QA sweep? Thanks!

**Comment by Samara-Strauss** [comment]
Just checked these updates in staging and all looks good!

**Pull request merged**